### ISSUE TITLE
azurerm_windows_web_app - 32_bit_worker - Invalid HCL Syntax

### ISSUE THREAD

**Issue by joewashear007** [hashicorp/terraform-provider-azurerm#15809]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Terraform (and AzureRM Provider) Version

* Terraform Core version: v1.1.7
* AzureRM Provider version:  v2.98.0

### Terraform Configuration Files

```hcl
resource "azurerm_windows_web_app" "app" {
  name                    = lower(var.name)
  location                = var.location
  resource_group_name     = var.resource_group_name
  app_service_plan_id     = var.app_service_plan_id
  https_only              = true
  client_affinity_enabled = true
  app_settings            = merge(local.app_settings, var.app_settings)
  tags                    = merge(local.tags, var.tags)
  identity {
    type = "SystemAssigned"
  }

  site_config {
    http2_enabled             = true
    vnet_route_all_enabled    = true
    always_on                 = true
    ftps_state                = "Disabled"
    32_bit_worker             = var.use_32_bit
    health_check_path         = var.health_check_path
    default_documents         = concat(var.default_documents, ["Default.htm", "Default.html", "Default.asp", "index.htm", "index.html", "iisstart.htm", "default.aspx", "index.php", "hostingstart.html"])

    application_stack {
      current_stack = "dotnet"
      dotnet_version = "v4.0"
    }
  }
}
```

### Description / Feedback
```
cquiring state lock. This may take a few moments...
Releasing state lock. This may take a few moments...
╷
│ Error: Argument or block definition required
│
│   on ..\..\..\..\modules\web_app\resource.tf line 26, in resource "azurerm_windows_web_app" "app":
│   26:     32_bit_worker             = var.use_32_bit
│
│ An argument or block definition is required here.
╵
```
I am getting syntax errors with the  `SiteConfig.32_bit_worker` property. Terraform doesn't like that the property begins with a number. Is there a work around for the syntax? The terraform HCL language says the identifiers shouldn't start with a number. 
### References

 - https://www.terraform.io/language/syntax/configuration#identifiers

**Comment by jackofallops** [comment]
Hi @joewashear007 - Thanks for this, looks like I missed a docs update when renaming that property. It should be `use_32_bit_worker` not `32_bit_worker ` - I'll PR a docs fix in shortly.

**Comment by jackofallops** [comment]
Closed by #15912
